packages/ozone/src/mod-service/index.ts

```diff
@@ -858,12 +857,14 @@ export class ModerationService {
       neg: !!l.neg,
     }))
     const { ref } = this.db.db.dynamic
+    await sql`notify ${ref(LabelChannel)}`.execute(this.db.db) // emitted transactionally
     const excluded = (col: string) => ref(`excluded.${col}`)
     await this.db.db
       .insertInto('label')
       .values(dbVals)
       .onConflict((oc) =>
         oc.columns(['src', 'uri', 'cid', 'val']).doUpdateSet({
+          id: sql`${excluded('id')}`,
```

Comment:
Interesting!  Slightly surprised this just works, cool.

Reply:
Yeah i was unsure if it was going to. But seems to work nicely & thinking it through, it make sense since the ID is taken from the sequence before the write actually occurs